package.json

```diff
@@ -133,6 +133,7 @@
     "react-use-css-breakpoints": "^1.0.3",
     "resize-observer-polyfill": "^1.5.1",
     "screenfull": "^4.0.1",
+    "sdp-transform": "^2.14.1",
```

Comment:
No change in package-lock.json should be needed because this is the version already being used by mediasoup. Hard to tell for sure because my local NPM rewrote the entire file.